app/views/search/_collection_details.html.erb

```diff
@@ -133,11 +133,11 @@
         </div>
         <!-- end of first row -->
 
-        <!-- second row: metadata links | api endpionts | relative urls-->
+        <!-- second row: metadata links | api endpoints | relative urls-->
         <div class="row">
           <div class="flex-column">
             <!-- description -->
-            <div class="long-paragraph collapsed" data-bind="text: $data.description"></div>
+            <div class="long-paragraph collapsed" data-bind="descriptionLinks: $data.description "></div>
```

Comment:
Nice and clean -- I like this! :) Since there is another change here anyway, can we remove the extra space at the end of this data-binding?